package.json

```diff
@@ -75,8 +75,9 @@
 		}
 	},
 	"dependencies": {
-		"@automattic/calypso-build": "1.0.0",
+		"@automattic/calypso-build": "4.0.1",
```

Comment:
Updating the build tool is likely easier and safer to do in its own PR?

Reply:
Working on this in https://github.com/Automattic/jetpack/pull/13503

Reply:
#13503 shipped.